test/fixtures/markup/missing_closing_brace_after_self_close_on_close.mf2

```diff
@@ -11,6 +11,10 @@ Markup tag can not be self-closing if it is a close tag. (at @6..7)
 Markup tag is not closed with a closing brace. (at @0..7)
```

Comment:
Can we have a quick fix for this?

Reply:
Also for expressions and quoted patterns?